src/main/java/org/jenkinsci/plugins/xunit/XUnitDefaultValues.java

```diff
@@ -42,6 +42,10 @@ public class XUnitDefaultValues {
 
     public static final String JUNIT_FILE_PREFIX = "TEST-";
 
-    public static final String JUNIT_FILE_EXTENSION = ".xml";
+    public static final String JUNIT_FILE_EXTENSION = ".xml";    
+
+    public static final String DEFAULT_CHECKS_NAME = "Tests";
+
+    public static final boolean SKIP_PUBLISHING_CHECKS = false;
```

Comment:
for backward compatibility default to true (feature disabled) as in config.jelly

Reply:
In [junit-plugin](https://github.com/jenkinsci/junit-plugin) (https://github.com/jenkinsci/junit-plugin/pull/180) the change was also introduced as "breaking change". It would only lead to people seeing their test results in GitHub. I can change it to being disabled by default and update the documentation (`README.md`) accordingly.

Reply:
this plugin is mainly used by private corporation that could not use github. So it's better be safe and in case they will configure on needs.

Reply:
If the [GitHub Checks Plugin](https://github.com/jenkinsci/github-checks-plugin) is not installed nothing really happens besides having the overhead of constructing some strings and getting the log message `[Checks API] No suitable checks publisher found.` inside the console. 
Btw the `[Checks API] No suitable checks publisher found.` log message is the only reason i know that this feature exists.